src/main/java/appeng/client/gui/implementations/GuiInterfaceTerminal.java

```diff
@@ -162,12 +286,19 @@ protected void mouseClicked( final int xCoord, final int yCoord, final int btn )
 	public void drawBG( final int offsetX, final int offsetY, final int mouseX, final int mouseY )
 	{
 		this.bindTexture( "guis/interfaceterminal.png" );
-		this.drawTexturedModalRect( offsetX, offsetY, 0, 0, this.xSize, this.ySize );
+		this.drawTexturedModalRect( offsetX, offsetY, 0, 0, this.xSize, 18 );
+
+		for( int x = 0; x < rows; x++ )
+		{
+			this.drawTexturedModalRect( offsetX, offsetY + 18 + x * SECTION_HEIGHT, 0, 18, this.xSize, SECTION_HEIGHT);
+		}
+
+		this.drawTexturedModalRect( offsetX, offsetY + 18 + rows * SECTION_HEIGHT - 2, 0, 124, this.xSize, 96);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)